### PR TITLE
Add AnayaDesign's hometown

### PR DESCRIPTION
**Pull request by anayadesign**
Adding Santa Rosa, CA to the hometown list 👍 
🍻 🍖 🏒 

@garyjzhao please review!

### CODE REVIEW

**Comment by garyjzhao** [comment]
👍

File: anayadesign-santarosa.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+- Russian River Brewery
+- Stark's Steakhouse
+- Hank's Creekside
+
+### Things to do
+- Hike around Spring Lake
+- Snoopy's Home Ice
+- Third street aleworks
```
Comment (garyjzhao) [inline anayadesign-santarosa.md:11]: You forgot the local 7-Eleven.

**Review by garyjzhao — CHANGES_REQUESTED** [review]
Please add some convenience stores

File: anayadesign-santarosa.md (hunk 1 above)
Comment (anayadesign) [reply anayadesign-santarosa.md:11]: Daiso is better

**Review by garyjzhao — APPROVED** [review]

**Pull request merged**